src/test/java/gov/hhs/cms/bluebutton/fhirstress/utils/FhirClientTest.java

```diff
@@ -0,0 +1,47 @@
+package gov.hhs.cms.bluebutton.fhirstress.utils;
+
+import junit.framework.Test;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+import org.hl7.fhir.dstu3.model.Patient;
+import ca.uhn.fhir.rest.client.IGenericClient;
+import ca.uhn.fhir.model.primitive.UriDt;
+
+/**
+ * Unit test for simple FhirClient.
+ */
+public class FhirClientTest 
```

Comment:
This relies on an external FHIR sever and so should be renamed to `FhirClientIT` and run by `failsafe` instead of `surefire`.

Reply:
Done.